pygen/pygen_src/README.md

```diff
@@ -0,0 +1,74 @@
+## Overview
+
+RISCV-DV-PyFlow is a python based open-source instruction generator for RISC-V
+processor verification. It currently supports the following features:
+
+- Supported instruction set: RV32IMAFDC
+- Supported privileged mode: machine mode
+- Illegal instruction and HINT instruction generation
+- Random forward/backward branch instructions
+- Supports mixing directed instructions with random instruction stream
+- Support for direct & vectored interrupt table.
+- Multi-hart support
+- Instruction generation coverage model(Currently supports RV32I and RV32M only)
```

Comment:
Functional coverage framework (reports GUI as well as text, currently supports RV32I and RV32M only)